CONTRIBUTING.md

```diff
@@ -7,7 +7,11 @@ Follow these steps for local or [cloud-IDE][] (via [Gitpod.io][]) development:
 - Fork and clone this repository (for local development only).
 - Install the latest [LTS release][] of **Node**, using **[nvm][]** for example:
   ```console
-  $ nvm install --lts
```

Comment:
The original command invocation is correct according to the `nvm` help info and docs.